requirements-devel.txt

```diff
@@ -1,4 +1,5 @@
 requests>=2.30.0,<3.0.0
+httpx>=0.25.0,<1.0.0
 validators>=0.21.2,<1.0.0
```

Comment:
I think it would be prudent to pin this to dep to `0.25.0` since `httpx` is not `v1` yet, wdyt? The same could be said for the `validators` dep also